docs/articles/actors/dispatchers.md

```diff
@@ -184,6 +184,9 @@ The `ChannelExectuor` re-uses the same threading settings as the `ForkJoinExecut
 ```hocon
 akka.actor.default-dispatcher = {
     executor = channel-executor
+    # channel executor priority to use
+    # - available: idle, low, normal, high
+    channel-executor.priority = normal
     fork-join-executor { #channelexecutor will re-use these settings
```

Comment:
These are not used by channel executor anymore, right?

Reply:
Can't move forward until this is resolved @eaba @Zetanova

Reply:
`channel-executor.priority` is used by channel executor and the only setting for the `channel-executor`

Reply:
no, I meant the `fork-join-executor` configuration comment